NeXusStandard.tex

```diff
@@ -0,0 +1,56 @@
+NeXus is an effort by an international group of scientists to define a common data exchange and archival format for neutron, X-ray and muon experiments. NeXus is built on top of the scientific data format HDF5 and adds domain-specific rules for organizing data within HDF5 files.
```

Comment:
The first sentence is probably already covered in the introduction.
Worth mentioning that HDF5 files can be read and written using the h5py library and is used by other popular Python libraries such as pandas.